Inc/HALAL/Services/Communication/SPI/SPI.hpp

```diff
@@ -19,6 +19,8 @@ extern SPI_HandleTypeDef hspi3;
  * 
  */
 class SPI{
+public:
```

Comment:
```suggestion
public:
```
Porque hay un public si no hay nada dentro?

Reply:
Ahora lo arreglo.